dev/Pager/PrototypePager/Scripts/PageChangedEventArgs.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Windows.UI.Xaml;
+
+namespace MUXControlsTestApp
+{
+    public sealed class PageChangedEventArgs : EventArgs
+    {
+        public int CurrentPage { get; private set; }
+
+        public PageChangedEventArgs(int NewIndex)
+        {
+            CurrentPage = NewIndex;
```

Comment:
We should run this by Marissa, but I think a previousPage property is probably warrented

Reply:
I added in the property